code/Attack.ipynb

```diff
@@ -0,0 +1,1471 @@
+{
```

Comment:
<p>CLAW reference missing. </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/sayakpaul/robustness-foundation-models/pull/1/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='27'/>

Reply:
<p>I haven't put this paper on the arxiv, will put it on later. Then I will add the reference .</p>

Reply:
Then at least leave a placeholder so that we don't forget.